src/redux/books/boooks-redux.js

```diff
@@ -0,0 +1,28 @@
+const ADD_BOOK = 'bookstore/src/redux/books/addBook';
+const REMOVE_BOOK = 'bookstore/src/redux/books/removeBook';
+
+// action to add
+const addBook = () => ({
+  type: ADD_BOOK,
+  payload: test,
+});
+
+// action to delete
+const removeBook = () => ({
+  type: REMOVE_BOOK,
+  payload: test,
+});
+
+// create a reducer
+const initialState = (state = '', action) => {
+  if (action.type === ADD_BOOK) {
+    return action.payload;
```

Comment:
- [ ] As required, in the case of where `action.type` is `ADD_BOOK` the function should return a new state object in which the books array will contain a new book object, passed by action.payload. You're not returning a new state below. See a Correct Example below:

```
  case ADD_BOOK:
      return [
        ...state,
        action.payload,
      ];

```